lib/pure/random.nim

```diff
@@ -115,6 +115,10 @@ proc rand*(r: var Rand; max: int): int {.benign.} =
   ## random number is always the same, unless `randomize` is called
   ## which initializes the random number generator with a "random"
   ## number, i.e. a tickcount.
+  when not defined(release):
```

Comment:
compileOption("rangeChecks") would work better here

Reply:
Sure, didn't know about that. Do I just `/not defined(release)/compileOption("rangeChecks")/g`?

Reply:
I suppose

Reply:
Cool, thank you